tests/unit/project_api/test_project_api.py

```diff
@@ -219,3 +236,150 @@ def test_get_workgroup_review_items(self, reviewer_id, content_id, review_items,
+
+        self.assert_project_data(project1, canned_responses.Projects.project1)
+        self.assert_project_data(project2, canned_responses.Projects.project2)
+
+    @ddt.data(
+        ('course1', 'content1'),
+        ('course2', 'content2'),
+    )
+    @ddt.unpack
+    def test_get_project_by_content_id(self, course_id, content_id):
+        expected_parameters = {
+            'course_id': course_id,
+            'content_id': content_id
+        }
+        calls_and_results = {(PROJECTS_API,): [canned_responses.Projects.project1]}
+
+        with self._patch_send_request(calls_and_results) as patched_send_request:
+            project = self.project_api.get_project_by_content_id(course_id, content_id)
+            self.assert_project_data(project, canned_responses.Projects.project1)
+
+            patched_send_request.assert_called_once_with(GET, (PROJECTS_API,), query_params=expected_parameters)
+
+    def test_get_project_by_content_id_fail_if_more_than_one(self):
+        calls_and_results = {
+            (PROJECTS_API,): [canned_responses.Projects.project1, canned_responses.Projects.project2]
+        }
+        with self._patch_send_request(calls_and_results), \
+                self.assertRaises(AssertionError):
+            self.project_api.get_project_by_content_id('irrelevant', 'irrelevant')
+
+    def test_get_project_by_content_id_return_none_if_not_found(self):
+        calls_and_results = {(PROJECTS_API,): []}
+        with self._patch_send_request(calls_and_results):
+            project = self.project_api.get_project_by_content_id('irrelevant', 'irrelevant')
+            self.assertIsNone(project)
+
+    @ddt.data(
+        (1, canned_responses.Workgroups.workgroup1),
+        (2, canned_responses.Workgroups.workgroup2),
+    )
+    @ddt.unpack
+    def test_get_workgroup_by_id(self, group_id, expected_result):
+        calls_and_results = {
+            (WORKGROUP_API, 1): canned_responses.Workgroups.workgroup1,
+            (WORKGROUP_API, 2): canned_responses.Workgroups.workgroup2
+        }
+
+        with self._patch_send_request(calls_and_results) as patched_send_request:
+            workgroup = self.project_api.get_workgroup_by_id(group_id)
+            patched_send_request.assert_called_once_with(GET, (WORKGROUP_API, group_id))
+
+        self.assertEqual(workgroup.id, expected_result['id'])
+        self.assertEqual(workgroup.project, expected_result['project'])
+        self.assertEqual(len(workgroup.users), len(expected_result['users']))
+        self.assertEqual([user.id for user in workgroup.users], [user['id'] for user in expected_result['users']])
+
+    @ddt.data(
+        ('course1', 'content1'),
+        ('course1', 'content2'),
+        ('course2', 'content3')
+    )
+    @ddt.unpack
+    def test_get_completions_by_content_id(self, course_id, content_id):
+        def build_url(course_id, content_id):
+            return self.project_api._build_url(  # pylint:disable=protected-access
```

Comment:
Should this method's name be prefixed with an underscore if we are intending to use it outside of its own class?